### PR TITLE
Add testing upgrade guide

### PR DESCRIPTION
**Pull request by timbussmann**
documents the breaking changes made by https://github.com/Particular/NServiceBus.Testing/issues/100

### CODE REVIEW

**Comment by weralabaj** [comment]
@timbussmann I know it's a WIP, but remember about adding it in the menu :)

File: nservicebus/upgrades/testing-6to7.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ - 6
+ - 7
+---
+
+
+## Upgrade to NServiceBus Version 7
+
+NServiceBus.Testing requires NServiceBus Version 7.
+
+As part of upgrading to NServiceBus.Testing Version 7, projects will also require an upgrade to [NServiceBus Version 6](/nservicebus/upgrades/6to7/).
```
Comment (bording) [inline nservicebus/upgrades/testing-6to7.md]: Shouldn't the link text be `NServiceBus Version 7` and not `NServiceBus Version 6` ?

File: nservicebus/upgrades/testing-6to7.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### AssertSagaCompletionIs
+
+The `AssertSagaCompletionIs` method has been obsoleted and replaced by `ExpectSagaCompleted` and `ExpectSagaNotCompleted`. Instead of:
+
+    Test.Saga<MySaga>()
+        .WhenHandling<TestSaga.CompleteSagaMessage>()
+        .AssertSagaCompletionIs(true);
+        
+use the new expecations to write:
```
Comment (bording) [inline nservicebus/upgrades/testing-6to7.md]: ~~expecations~~ expectations

File: nservicebus/upgrades/testing-6to7.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    Test.Saga<MySaga>()
+        .ExpectSagaCompleted()
+        .WhenHandling<TestSaga.CompleteSagaMessage>();
+        
+Note how the `ExpectSagaCompleted` and `ExpectSagaNotCompleted` expectations must be placed **before** a `When` method, similar to other expecation methods.
```
Comment (bording) [inline nservicebus/upgrades/testing-6to7.md]: ~~expecation~~ expectation

**Pull request merged**